### DISCUSSION TITLE
Intercept packets before sending them to players

### DISCUSSION THREAD

**Discussion by MRtecno98** [Minestom/Minestom#1075] [Help] [answered]
I need to intercept [SetSlot packets](https://wiki.vg/Protocol#Set_Slot) to edit item data based on the player that views said item, is this practical to do in minestom? I know that there is a  `PlayerPacketOutEvent` but it cannot be cancelled nor the packet can be changed in the event itself so it's not really useful

**Comment by cody-quinn** [comment, marked as answer]
Sadly this is not currently possible.

My solution to this has been to just send a new packet during the PlayerPacketOutEvent, it will be sent after the first one.

**Reply by MRtecno98** [reply]
Yeah but i'd need to cancel the packet currently being sent and PlayerPacketOutEvent isn't cancellable. Too bad
